lib/core/src/lib/rich-text-editor/rich-text-editor.component.stories.ts

```diff
@@ -0,0 +1,123 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { RichTextEditorModule } from './rich-text-editor.module';
+import { Meta, moduleMetadata, Story } from '@storybook/angular';
+import { CoreStoryModule } from '../testing/core.story.module';
+import { RichTextEditorComponent } from './rich-text-editor.component';
+
+const exampleData = {
```

Comment:
A good practice is to put all the mock data in a separate mock folder. Here's an example 🤠: 
![image](https://user-images.githubusercontent.com/64163335/191505508-8d55a943-e105-49e3-9322-f33e96142240.png)
![image](https://user-images.githubusercontent.com/64163335/191505548-9f442f37-8493-4092-95c9-885ce2eadd10.png)

Reply:
done :)